### PR TITLE
added email confirmation modal templates

### PR DESCRIPTION
**Pull request by greentoss**
please check if im doing it right

i have created 2 templates for modals email confirmation or rejection

### CODE REVIEW

File: src/components/modals/email-confirm-modal/email-confirm-modal.js [hunk 1]
```diff
@@ -0,0 +1,73 @@
+import { Button, Dialog, IconButton } from '@mui/material'
```
Comment (greentoss) [inline src/components/modals/email-confirm-modal/email-confirm-modal.js:1]: please check if im doing it right

i have created 2 templates for modals email confirmation or rejection

File: src/components/modals/email-reject-modal/email-reject-modal.js [hunk 2]
```diff
@@ -0,0 +1,73 @@
+import { Button, Dialog, IconButton } from '@mui/material'
+import Box from '@mui/material/Box'
+import Typography from '@mui/material/Typography'
+import CloseIcon from '@mui/icons-material/Close'
+import { style } from '../email-confirmation-modal.style'
+import { useState } from 'react'
```
Comment (tarasSanotskyi) [inline src/components/modals/email-reject-modal/email-reject-modal.js:6]: sort imports

File: src/components/modals/email-confirmation-modal.style.js [hunk 3]
```diff
@@ -0,0 +1,19 @@
+export const style = {
+  box: {
+    margin: { xs: '0 auto', sm: 0 },
+    padding: { xs: 0, sm: 2, md: 7 },
+    textAlign: 'center'
+  },
+  button: {
+    marginTop: '30px',
+    backgroundColor: '#262738',
```
Comment (tarasSanotskyi) [inline src/components/modals/email-confirmation-modal.style.js:9]: use colors from palette

**Review by rostyslaw27 — COMMENTED** [review]
do not delete package-lock.json

**Review by rostyslaw27 — COMMENTED** [review]
use better names for svgs

File: src/components/modals/email-reject-modal/email-reject-modal.js [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+import img from '~/assets/img/email-confirmation-modals/icons8-forbidden-circle-50.svg'
+import { styles } from '~/pages/error/styles/bad-request.styles'
+
+const EmailRejectModal = () => {
+  const [open, setOpen] = useState(true)
+
+  const handleClickOpen = () => {
```
Comment (tarasSanotskyi) [inline src/components/modals/email-reject-modal/email-reject-modal.js:13]: remove "handle"

File: src/components/modals/email-reject-modal/email-reject-modal.js [hunk 5]
```diff
@@ -0,0 +1,73 @@
...
+    setOpen(true)
+  }
+
+  const handleClose = () => {
```
Comment (tarasSanotskyi) [inline src/components/modals/email-reject-modal/email-reject-modal.js:17]: same

File: src/components/modals/email-confirm-modal/email-confirm-modal.js [hunk 6]
```diff
@@ -0,0 +1,73 @@
...
+import Box from '@mui/material/Box'
+import Typography from '@mui/material/Typography'
+import CloseIcon from '@mui/icons-material/Close'
+import { style } from '../email-confirmation-modal.style'
+import { useState } from 'react'
+import img from '~/assets/img/email-confirmation-modals/icons8-check-circle-50.svg'
+import { styles } from '~/pages/error/styles/bad-request.styles'
+
+const EmailConfirmModal = () => {
+  const [open, setOpen] = useState(true)
```
Comment (rostyslaw27) [inline src/components/modals/email-confirm-modal/email-confirm-modal.js:11]: you don't need to create more modals. We already have a PopupDialog and need to pass content using a setModal

**Review by rostyslaw27 — CHANGES_REQUESTED** [review]

File: src/components/modals/email-confirm-modal/email-confirm-modal.js [hunk 7]
```diff
@@ -0,0 +1,73 @@
...
+    >
+      <Box sx={ style.box }>
+        <IconButton
+          aria-label="close"
+          onClick={ handleClose }
+          sx={ {
+            position: 'absolute',
+            right: 8,
+            top: 8,
+            color: (theme) => theme.palette.grey[500],
```
Comment (tarasSanotskyi) [inline src/components/modals/email-confirm-modal/email-confirm-modal.js:43]: color: 'grey.500'

**Review by tarasSanotskyi — CHANGES_REQUESTED** [review]

File: src/components/modals/email-confirmation-modal.style.js (hunk 3 above)
Comment (greentoss) [reply src/components/modals/email-confirmation-modal.style.js:9]: but what if color from figma #262738 does not presented in palette ?

File: src/components/modals/email-reject-modal/email-reject-modal.js (hunk 4 above)
Comment (greentoss) [reply src/components/modals/email-reject-modal/email-reject-modal.js:13]: done

File: src/components/modals/email-reject-modal/email-reject-modal.js (hunk 5 above)
Comment (greentoss) [reply src/components/modals/email-reject-modal/email-reject-modal.js:17]: done

File: src/components/modals/email-confirm-modal/email-confirm-modal.js (hunk 6 above)
Comment (greentoss) [reply src/components/modals/email-confirm-modal/email-confirm-modal.js:18]: used modal in GuestHomePage

**Pull request closed without merging**